03-use-cases/01-finance/portfolio-management/cvar-optimizer/README.md

```diff
@@ -6,11 +6,8 @@
 
 Therefore, run the following in this directory:
 
-1. `poetry install`
-2. `poetry run python -m ipykernel install --user --name=cvar-optimizer`
-3. `poetry run jupyter-lab`
-
-Finally, select the `cvar-optimizer` kernel in JupyterLab to run this notebook.
+1. `uv sync --python 3.10`
```

Comment:
This used to be `uv pip install -r pyproject.toml` but this was the wrong move. We want to have users install via our version pinned lockfile, not have them resolve dependencies themselves and create their own (potentially different) lockfiles. Hence the change to `uv sync --python 3.10`